battleship-client/src/test/java/com/academy/solid/nie/client/ui/BoardTest.java

```diff
@@ -24,7 +23,7 @@ public void shouldPlaceShip_whenThereAreNoShips() {
         EventHandler<MouseEvent> handler = Mockito.mock(EventHandler.class);
         board.initialize(handler);
 
-        new Cell(point).addShip(ship);
+        new Cell(point).setShip(ship);
 
         //when
         boolean result = board.isShipPositionValid(ship);
```

Comment:
Result is a weak name here, it makes your assertion useless. When this fails I'll just learn that result wasn't true. What was the result? I won't know without looking in the code.

Reply:
should we inline this? or rename to isShipPositionValid?

Reply:
Inline wins for me. 

```
// when - then

assertTrue(board.isShipPositionValid(ship);
```